index.template.html

```diff
@@ -1194,25 +1210,25 @@ <h3>Default Value Definitions</h3>
                 <td>
                     <code>op</code>
                 </td>
-                <td>Array of strings with values <code>readproperty</code> followed by 
-                    <code>writeproperty</code></td>
-                <td>if defined within an instance of <code>PropertyAffordance</code></td>
+                <td><a>Array</a> of <code>string</code> with the element <code>readproperty</code> followed by <code>writeproperty</code>
```

Comment:
Actually, we don't care about the order. Would simplify to:

<a>Array</a> of <code>string</code> with the elements <code>readproperty</code> and <code>writeproperty</code>

Best make a line break after <td>